enterprise-suite/templates/prometheus.yaml

```diff
@@ -158,6 +158,19 @@ spec:
               scheme: HTTP
             initialDelaySeconds: 30
 
+        - name: es-monitor-api-configmap-reload
+          image: jimmidyson/configmap-reload:{{ .Values.configMapReloadVersion }}
+          resources:
+            requests:
+              cpu: {{ default .Values.defaultCPURequest .Values.configmapReloadCPURequest }}
+              memory: {{ default .Values.defaultMemoryRequest .Values.configmapReloadMemoryRequest }}
+          args:
+          - --volume-dir=/monitor-data
+          - --webhook-url=http://localhost:8180/reload
```

Comment:
`127.0.0.1` instead of `localhost` would avoid a host lookup, not that we expect that to matter in this environment, but we've seen customers with non-kubernetes environments where localhost lookup was fubar.

Reply:
Agreed, I've also seen places erroneously map `localhost` to actual IPs.